src/views/new.ecr

```diff
@@ -7,9 +7,22 @@
 <main>
   <section id="form-new">
     <script id="script-lib">
```

Comment:
My current implementation stores the file as json:

```js
{
   name: "image.png"
   content: "base64EncodedFileContent"
}
```
and then does the same as for plaintext. This will result in base64 in json in encryption in base64. Which does make it more complex to understand (and therefore makes seme.li worse). 

Also I don't understand why the `ciphertext` is converted to be base64.